migrations/versions/2cebdd533a59_.py

```diff
@@ -0,0 +1,40 @@
+"""empty message
+
+Revision ID: 2cebdd533a59
+Revises: 0bd6fd83b781
+Create Date: 2024-10-24 13:52:16.788586
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '2cebdd533a59'
+down_revision = '0bd6fd83b781'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('jira_tasks', schema=None) as batch_op:
+        batch_op.add_column(sa.Column('summary', sa.String(), nullable=True))
+        batch_op.alter_column('created_at',
```

Comment:
I was worried about this, but it's right since timestamp is a superclass of datetime and can be substituted by it

Reply:
it's working well now after this migration